pilot/cmd/pilot-agent/main.go

```diff
@@ -519,15 +519,16 @@ var (
 // dedupes the string array and also ignores the empty string.
 func dedupeStrings(in []string) []string {
 	stringMap := map[string]bool{}
+	unique := make([]string, 0)
```

Comment:
make([]string, 0, len(in))

Reply:
no. this is wrong, you can't define by this way :make([]string, 0, len(in))
we use append here.
unique = append(unique, c)

Reply:
accept